document-store/build.gradle.kts

```diff
@@ -6,99 +6,99 @@ import com.bmuschko.gradle.docker.tasks.network.DockerCreateNetwork
 import com.bmuschko.gradle.docker.tasks.network.DockerRemoveNetwork
 
 plugins {
-  `java-library`
-  jacoco
-  id("org.hypertrace.publish-plugin")
-  id("org.hypertrace.jacoco-report-plugin") version "0.1.0"
-  id("org.hypertrace.integration-test-plugin") version "0.1.0"
-  id("com.bmuschko.docker-remote-api") version "6.4.0"
+    `java-library`
```

Comment:
Is this to verify the check is working? The indentation should be 2 https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Reply:
We need to fix all files before merge so next person opening a PR in here does not have to deal with the lint mess.

Reply:
Agreed - but aren't these formatting changes going in the wrong direction? They're changing the formatting of files that were previously correctly formatted and after this change are incorrect. As an example, take Datastore.java - https://github.com/hypertrace/document-store/pull/24/files#diff-fcb7b4dcc9273e596438f61590e9eb81e373bec38fa40adec47e2068853fcd3a

The changes switch the block indent from 2 -> 4 (should be 2 https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation ) and separate the lang imports from third party (should be in single block https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing )

Reply:
Yeah, let me rephrase. This was an attempt to fix all problems but I was fooled by the intellij plugin I think.